samples/highcharts/plotoptions/series-marker-enabled/demo.js

```diff
@@ -1,17 +1,28 @@
-Highcharts.chart('container', {
+Highcharts.chart("container", {
     xAxis: {
-        categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+        categories: [
+            "Jan", "Feb", "Mar", "Apr", "May", "Jun",
+            "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
+        ]
     },
 
     plotOptions: {
         series: {
             marker: {
                 enabled: false
+            },
+            states: {
+                hover: {
+                    enabled: false
+                }
             }
         }
     },
 
     series: [{
-        data: [29.9, 71.5, 106.4, 129.2, 144.0, 176.0, 135.6, 148.5, 216.4, 194.1, 95.6, 54.4]
+        data: [
+            29.9, 71.5, 106.4, 129.2, 144.0, 176.0,
+            135.6, 148.5, 216.4, 194.1, 95.6, 54.4
```

Comment:
I'm not sure about this line break in the middle of array. It looks nice for our brains, but it's not a common rule.
Personally, as it's only a jsFiddle demo, I'd keep it in one line. Breaking into 12+2 lines doesn't make sense imho.

@TorsteinHonsi could you decide so we have one rule and stick to it in the future?

Reply:
@raf18seb  I prefer breaking up the array at 80 characters because the alternatives are worse. There are basically three options.

* No break. This may cause horizontal scrolling in jsFiddle, code editors and in code previews.
```js
// bad
data: [29.9, 71.5, 106.4, 129.2, 144.0, 176.0, 135.6, 148.5, 216.4, 194.1, 95.6, 54.4]
```

* Break for all items. This will require too much vertical space, at least for single values
```js
// bad
data: [
  29.9,
  71.5,
  106.4,
  129.2,
  144.0,
  176.0,
  135.6,
  148.5,
  216.4,
  194.1,
  95.6,
  54.4
]
```

* Compromise, break at 80
```js
// good
data: [
   29.9, 71.5, 106.4, 129.2, 144.0, 176.0, 135.6, 148.5, 216.4,
   194.1, 95.6, 54.4
]
```